SensusService/Probes/PollingProbe.cs

```diff
@@ -208,7 +237,33 @@ protected override void InternalStart()
 
                     }, GetType().FullName + " Poll", TimeSpan.FromMinutes(_pollingTimeoutMinutes), userNotificationMessage);
 
+#if __IOS__
+                if (_significantChangePoll)
+                {
+                    _locationManager.RequestAlwaysAuthorization();
+                    _locationManager.DistanceFilter = 5.0;
+                    _locationManager.PausesLocationUpdatesAutomatically = true;
+                    _locationManager.AllowsBackgroundLocationUpdates = true;
+
+                    if (CLLocationManager.LocationServicesEnabled)
+                    {
+                        _locationManager.StartMonitoringSignificantLocationChanges();
+                    }
+                    else
+                    {
+                        SensusServiceHelper.Get().Logger.Log("Location services not enabled; please enable in Settings.", LoggingLevel.Normal, GetType());
+                    }
+
+                    _locationManager.LocationsUpdated += (o, e) => SensusServiceHelper.Get().ScheduleOneTimeCallback(callback, 0);
```

Comment:
We get significant change polls when the app is inactive if the phone is moving, just not if it's stationary. If this were the problem, it sounds like significant change should do nothing in either case until the app is foregrounded; am I misunderstanding or does this sound correct?